CONTRIBUTING.md

```diff
@@ -229,6 +236,49 @@ add_holohub_application(my_application DEPENDS
 Refer to the [HoloHub application template folder](./applications/template/) for stub `metadata.json` and `README` files to copy
 and update for your new application.
 
+You can then build that application by configuring your build with `-D APP_my_application:BOOL=1` :
+
+```bash
+cmake -S . -B ./build -D APP_my_application:BOOL=1
+cmake --build ./build -j
+```
+
+### Adding a Packager
```

Comment:
Nitpick: "packager" implies some defined entity that is doing packaging, seems like CMake/CPack are the "packager"

```suggestion
### Adding a Package
```

Reply:
Yeah that's just semantics, but that's actually why I went for packager over package: contributors don't add a package (unlike how they actually add an app or operator source code), they add configurations to create a package, so I felt like packager was more appropriate? 

Or just `Adding a Package Configuration`?

Reply:
>Adding a Package Configuration

I like this one 👍